lms/djangoapps/certificates/migrations/0009_certificatetemplate_language.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('certificates', '0008_schema__remove_badges'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='certificatetemplate',
+            name='language',
+            field=models.CharField(blank=True, max_length=2, null=True, help_text='Optional. Only certificates for courses in the selected language will be rendered using this template.<br />*Note* Course languages are determined by the first two letters of the language code.', choices=[(b'en', 'English'), (b'es', 'Espa\xf1ol')]),
```

Comment:
To echo what I said in a comment below, I don't think we should include `choices` in this migration since this will be committed and we want to allow opensource users to have a different set of language choices.